ibm/resource_ibm_cbr_zone.go

```diff
@@ -0,0 +1,652 @@
+										Description: "The id of the account owning the service.",
+									},
+									"service_type": &schema.Schema{
+										Type:        schema.TypeString,
+										Optional:    true,
+										Description: "The service type.",
+									},
+									"service_name": &schema.Schema{
+										Type:        schema.TypeString,
+										Optional:    true,
+										Description: "The service name.",
+									},
+									"service_instance": &schema.Schema{
+										Type:        schema.TypeString,
+										Optional:    true,
+										Description: "The service instance.",
+									},
+								},
+							},
+						},
+					},
+				},
+			},
+			"excluded": &schema.Schema{
+				Type:        schema.TypeList,
+				Optional:    true,
+				Description: "The list of excluded addresses in the zone. Only addresses of type `ipAddress`, `ipRange`, and `subnet` can be excluded.",
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"type": &schema.Schema{
+							Type:        schema.TypeString,
+							Optional:    true,
+							Description: "The type of address.",
+						},
+						"value": &schema.Schema{
+							Type:        schema.TypeString,
+							Optional:    true,
+							Description: "The IP address.",
+						},
+						"ref": &schema.Schema{
+							Type:        schema.TypeList,
+							MaxItems:    1,
+							Optional:    true,
+							Description: "A service reference value.",
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"account_id": &schema.Schema{
+										Type:        schema.TypeString,
+										Computed:    true,
+										Description: "The id of the account owning the service.",
+									},
+									"service_type": &schema.Schema{
+										Type:        schema.TypeString,
+										Optional:    true,
+										Description: "The service type.",
+									},
+									"service_name": &schema.Schema{
+										Type:        schema.TypeString,
+										Optional:    true,
+										Description: "The service name.",
+									},
+									"service_instance": &schema.Schema{
+										Type:        schema.TypeString,
+										Optional:    true,
+										Description: "The service instance.",
+									},
+								},
+							},
+						},
+					},
+				},
+			},
+			"x_correlation_id": &schema.Schema{
+				Type:         schema.TypeString,
+				Optional:     true,
+				ValidateFunc: InvokeValidator("ibm_cbr_zone", "x_correlation_id"),
+				Description:  "The supplied or generated value of this header is logged for a request and repeated in a response header for the corresponding response. The same value is used for downstream requests and retries of those requests. If a value of this headers is not supplied in a request, the service generates a random (version 4) UUID.",
+			},
+			"transaction_id": &schema.Schema{
+				Type:         schema.TypeString,
+				Computed:     true,
+				ValidateFunc: InvokeValidator("ibm_cbr_zone", "transaction_id"),
```

Comment:
computed attributes are API responses..
Validations is not required for computed attributes

Reply:
removed